x-pack/plugins/security_solution/cypress/e2e/exceptions/shared_exception_lists_management/list_detail_page/list_details.cy.ts

```diff
@@ -91,4 +102,35 @@ describe('Exception list management page', () => {
     visitWithoutDateRange(exceptionsListDetailsUrl(getExceptionList1().list_id));
     cy.get(EXCEPTIONS_LIST_MANAGEMENT_DESCRIPTION).should('have.text', 'Add a description');
   });
+
+  it('Should create a new list and link it to two rules', () => {
+    createSharedExceptionList(
+      { name: 'Newly created list', description: 'This is my list.' },
+      true
+    );
+
+    // After creation - directed to list detail page
+    cy.get(EXCEPTIONS_LIST_MANAGEMENT_NAME).should('have.text', EXCEPTION_LIST_NAME);
+
+    // Open Link rules flyout
+    cy.get(EXCEPTION_LIST_DETAILS_LINK_RULES_BTN).click();
+
+    // Link the first Rule
+    clickOnLinkRulesByRuleRowOrderInListDetail(0);
+    // Link the second Rule
+    clickOnLinkRulesByRuleRowOrderInListDetail(1);
+
+    cy.get(MANAGE_RULES_SAVE).first().click();
+
+    cy.get(EXCEPTION_LIST_DETAIL_LINKED_TO_RULES_HEADER_MENU).should(
+      'have.text',
+      'Linked to 2 rules'
+    );
+    cy.get(EXCEPTION_LIST_DETAIL_LINKED_TO_RULES_HEADER_MENU).click();
```

Comment:
NIT: Wrap this into a task method.

Reply:
Done please check 5b877b615f33a075a985af1d642aad9d4287b90b